src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2937,6 +2937,11 @@ private bool IsMemberMissing(int member)
             return _lazyMakeMemberMissingMap != null && _lazyMakeMemberMissingMap.ContainsKey(member);
         }
 
+        internal void MakeTypeMissing(SpecialType type)
+        {
+            MakeTypeMissing((WellKnownType)type);
```

Comment:
>(WellKnownType)type [](start = 28, length = 19)

Please do not cast unrelated enums to each other. If you would like to share code, add private helpers that take an int and call it from both sets of overloads.

Reply:
Ah, sorry about that. I thought somehow `WellKnownTypes` were a superset of `SpecialTypes`.